pkg/apis/core/types.go

```diff
@@ -1499,10 +1499,14 @@ type KeyToPath struct {
 type LocalVolumeSource struct {
 	// The full path to the volume on the node.
 	// It can be either a directory or block device (disk, partition, ...).
-	// Directories can be represented only by PersistentVolume with VolumeMode=Filesystem.
-	// Block devices can be represented only by VolumeMode=Block, which also requires the
-	// BlockVolume alpha feature gate to be enabled.
 	Path string
+
+	// Filesystem type to mount.
+	// It applies only when the Path is a block device.
+	// Must be a filesystem type supported by the host operating system.
+	// Ex. "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
```

Comment:
Please change the documentation here (and v1)  to say that the default value is to auto-select a fileystem, rather than saying it is ext4.  That way we can actually change it later.

Reply:
should this be changed when we actually chane it ? Since now it still defaults to 'ext4'.
btw: some other plugins are saying like this too

Reply:
Let's change it now so we don't have to change it again later.  (and the new wording is still true to the implementation)

Reply:
ok, updated